import-scripts/generate_cdd_attribute_tsv.py

```diff
@@ -97,6 +99,15 @@ def create_uri_dictionary(uri_mapping_file):
             uri_dictionary[data[0].rstrip()] = data[1].rstrip()
     return uri_dictionary
 
+# Looks for non-ASCII characters
+# Uses 'isascii()' attribute for Java 3.7 and over
+def is_ASCII(s):
+    if(hasattr(s, 'isascii')): return s.isascii()
```

Comment:
please add a space after "if" .. and put the return clause on the following line (indented)